src/read/addr.rs

```diff
@@ -51,3 +51,52 @@ impl<R: Reader> From<R> for DebugAddr<R> {
         DebugAddr { section }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    extern crate test_assembler;
+
+    use self::test_assembler::{Endian, Label, LabelMaker, Section};
+    use super::*;
+    use read::EndianSlice;
+    use test_util::GimliSectionMethods;
+    use {Format, LittleEndian};
+
+    #[test]
+    fn test_get_address() {
+        for format in [Format::Dwarf32, Format::Dwarf64].iter().cloned() {
+            for address_size in [4, 8].iter().cloned() {
```

Comment:
Also -- great to exhaustively enumerate dwarf formats and address sizes like this!